### ISSUE TITLE
[docs] Provide examples in Typescript

### ISSUE THREAD

**Issue by dfreedm** [lit/lit-element#453]
It would be great to provide code examples in the guide in Typescript, particularly for users that want to compile with typescript's strict mode, or users of tslint

**Comment by hyperpress** [comment]
I'd second this and add we should consider adding a pwa-starter-kit example app written typescript that can showcase versatility.

**Comment by hyperpress** [comment]
Ah, I see. Yes, would be nice to have TS specific methods documented like #446 example.

**Comment by arthurevans** [comment]
I'm not sure what "Status: Pending" means but we're not doing active work on this ATM. This is effectively blocked by #550 (we want to have a mechanism for switching code samples, rather that supplying both TS and JS inline everywhere.

**Comment by sorvell** [comment]
Closing since [lit.dev ](https://lit.dev/) does this!
